source/ZividNETSamples.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.902
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30320.27
```

Comment:
What version of Visual Studio do you use? 2019?

Reply:
Yes, Im using 2019 now

Reply:
No, it is probably better to keep it to a constant version. I have changed it back to the original one (VS 2015)